### PR TITLE
NxTooltip Dark Mode - RSC-1053

### PR DESCRIPTION
**Pull request by EmilyBermudez88**
https://issues.sonatype.org/browse/RSC-1053

Currently the background colour of NxTooltip is incorrect in light mode. Ticket RSC-1493 (https://issues.sonatype.org/browse/RSC-1493) will deal with the variable change so the snapshots do not need to be updated in the dark mode ticket.

### CODE REVIEW

File: lib/src/base-styles/_nx-colors.scss [hunk 1]
```diff
@@ -76,5 +80,9 @@
     //NxBinaryDonutChart variables
     --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-30);
     --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-pink-80);
+
+      //NxTooltip variables
```
Comment (avnvbhatta) [inline lib/src/base-styles/_nx-colors.scss:84]:
```suggestion
    //NxTooltip variables
```

File: gallery/src/styles/CssVariables/CssVariablesPage.tsx [hunk 2]
```diff
@@ -349,6 +349,12 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-binary-donut-chart-arc-stroke">
               The color used on the <NxCode>NxBinaryDonutChart</NxCode> arc color.
             </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-tooltip-background">
+              The background color used on the <NxCode>NxTooltip</NxCode>.
+            </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-tooltip-text">
+              The color used on the <NxCode>NxTooltip</NxCode> text.
```
Comment (avnvbhatta) [inline gallery/src/styles/CssVariables/CssVariablesPage.tsx:356]:
```suggestion
              The text color used on the <NxCode>NxTooltip</NxCode>.
```

**Review by avnvbhatta — APPROVED** [review]

File: lib/src/base-styles/_nx-colors.scss [hunk 3]
```diff
@@ -55,6 +55,10 @@
   --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
   --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-teal-60);
 
+  //NxTooltip variables
+  --nx-color-tooltip-background: var(--nx-swatch-grey-05);
```
Comment (avnvbhatta) [inline lib/src/base-styles/_nx-colors.scss:59]: Suppose this needs to be `grey-20` as well?

Comment (EmilyBermudez88) [reply lib/src/base-styles/_nx-colors.scss]: > Suppose this needs to be `grey-20` as well?

Oops, I'd made the change locally and forgot to push! [9eb9c0c](https://github.com/sonatype/sonatype-react-shared-components/pull/974/commits/9eb9c0c04f607c4b3ab51325c151ad453eae1520)

**Review by avnvbhatta — APPROVED** [review]
LGTM!

**Review by JasonDSwearingen — APPROVED** [review]

**Pull request merged**